pom.xml

```diff
@@ -799,13 +799,30 @@ under the License.</licenseText>
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-pmd-plugin</artifactId>
+          <configuration>
+            <analysisCache>true</analysisCache>
+            <excludeFromFailureFile>.pmd/exclude.properties</excludeFromFailureFile>
+            <printFailingErrors>true</printFailingErrors>
+            <rulesets>
+              <ruleset>category/java/bestpractices.xml</ruleset>
```

Comment:
why would we not apply all java rulesets but only one?

Reply:
we can not pass even this one, so it might be overkill.